lib/rack/request.rb

```diff
@@ -501,23 +503,23 @@ def hostname
         end
       end
 
-      def strip_port(ip_address)
+      def strip_port(uri)
```

Comment:
`uri` is a misleading parameter name, it's really hostname + port.  As it doesn't appear that you are actually changing any logic in this method, I won't modify it at all.

Reply:
> uri is a misleading parameter name, it's really hostname + port.

Maybe.

> As it doesn't appear that you are actually changing any logic in this method, I won't modify it at all.

Now `strip_port` used for such data as `example.org`, and it doesn't look like `ip_address`.

Reply:
I've renamed to `hostname`. But I can revert this at all, if you're sure, but, again: domains don't look like IP addresses.

Reply:
I think it should be reverted.  `hostname` is more accurate (not completely as hostnames don't have ports), but not worth making git blame more difficult.

Reply:
OK, I've returned `ip_address` back. But, if you (Rack collaborators) will decide to rename it — I can.